### DISCUSSION TITLE
Peformance

### DISCUSSION THREAD

**Discussion by ghost** [PrismarineJS/mineflayer#2474] [Q&A]
A simple bot walking back and forth or doing anything simple how much ram and cpu does that take

**Comment by nickelpro** [comment]
There's nothing simple about walking back and forth, see several thousand lines of code that exist to make it work

**Reply by ghost** [reply]
Either you miss understood me or I wrote it wrong.
I meant running a simple example from here https://github.com/PrismarineJS/mineflayer/tree/master/examples
How much peformance will 1 bot consume

**Reply by amoraschi** [reply]
Try it and see
